app/controllers/api/subcollections/snapshots.rb

```diff
@@ -25,8 +23,7 @@ def snapshots_create_resource(parent, _type, _id, data)
       end
 
       def delete_resource_snapshots(parent, type, id, _data)
-        validation = parent.validate_remove_snapshot(id)
```

Comment:
`id` here is the id of the snapshot which is about to be deleted. You are correct, this must have been an error, but I guess a non-critical one?